subsys/bluetooth/host/gatt.c

```diff
@@ -2157,6 +2157,16 @@ static int gatt_notify_flush(struct bt_conn *conn)
 	return err;
 }
 
+static void gatt_notify_cleanup(struct bt_conn *conn)
```

Comment:
since it's static, maybe `cleanup_notify` instead?

Reply:
The `gatt_` prefix actually seems to be used quite extensively in `gatt.c` for static functions :) Personally I like when static functions are still prefix with some module name like that, as long as they don't use e.g. `bt_gatt_` :)

Reply:
I personally don't because that makes it harder to know whether they are public or internal. I need to revive the naming style PR...